Make `tasit-contracts` a dependency of other packages

The version `v0.0.11` of `tasit-sdk` is broking the `tasit` app.

a test output:
```sh
 FAIL  components/presentational/OnboardingHome.test.js
  ● Test suite failed to run

    Cannot find module '../../../tasit-contracts/abi/MyERC721Full.json' from 'ERC721Full.js'

    However, Jest was able to find:
    	'erc721/ERC721Full.js'

...
```

Direct import from filepath seems work inside of mono repo but lerna doesn't packing these files when publishing.
All of these imports should be move to package import and `tasit-contract` should be a dep of all packges that use it to get ABI or address of the contracts.
Those packages that are using it only for test purposes can have it as dev-dep only.

--- a/packages/tasit-action/package.json
+++ b/packages/tasit-action/package.json
@@ -26,7 +26,8 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.1.5",
-    "ethers": "^4.0.27"
+    "ethers": "^4.0.27",
+    "tasit-contracts": "^0.0.11"
   },
   "devDependencies": {
     "@babel/cli": "^7.1.5",

--- a/packages/tasit-action/src/contract/Contract.test.js
+++ b/packages/tasit-action/src/contract/Contract.test.js
@@ -1,6 +1,6 @@
 import Contract from "./Contract";
 import Account from "tasit-account";
-import TasitContracts from "../../../tasit-contracts/dist";
+import TasitContracts from "tasit-contracts";
 const { local: localContracts } = TasitContracts;
 const { SampleContract } = localContracts;
 const {

--- a/packages/tasit-action/src/erc20/ERC20Full.js
+++ b/packages/tasit-action/src/erc20/ERC20Full.js
@@ -1,5 +1,8 @@
 import Contract from "../contract/Contract";
-import abi from "../../../tasit-contracts/abi/MyERC20Full.json";
+import TasitContracts from "tasit-contracts";
+const { local } = TasitContracts;
+const { MyERC20Full } = local;
+const { abi } = MyERC20Full;
 
 export default class ERC20Full extends Contract {
   constructor(address, wallet) {

--- a/packages/tasit-action/src/erc20/ERC20Full.test.js
+++ b/packages/tasit-action/src/erc20/ERC20Full.test.js
@@ -1,5 +1,5 @@
 import ERC20Full from "./ERC20Full";
-import TasitContracts from "../../../tasit-contracts/dist";
+import TasitContracts from "tasit-contracts";
 const { local: localContracts } = TasitContracts;
 const { MyERC20Full } = localContracts;
 const { address: ERC20_FULL_ADDRESS } = MyERC20Full;

--- a/packages/tasit-action/src/erc20/Mana.js
+++ b/packages/tasit-action/src/erc20/Mana.js
@@ -1,9 +1,11 @@
 import Contract from "../contract/Contract";
-import manaTokenABI from "../../../tasit-contracts/abi/MANAToken.json";
+import TasitContracts from "tasit-contracts";
+const { local } = TasitContracts;
+const { MANAToken } = local;
+const { abi } = MANAToken;
 
 export default class Mana extends Contract {
   constructor(address, wallet) {
-    const abi = manaTokenABI;
     super(address, abi, wallet);
   }
 }

--- a/packages/tasit-action/src/erc20/Mana.test.js
+++ b/packages/tasit-action/src/erc20/Mana.test.js
@@ -1,5 +1,5 @@
 import Mana from "./Mana";
-import TasitContracts from "../../../tasit-contracts/dist";
+import TasitContracts from "tasit-contracts";
 const { local: localContracts } = TasitContracts;
 const { MANAToken } = localContracts;
 const { address: MANA_ADDRESS } = MANAToken;

--- a/packages/tasit-action/src/erc721/ERC721Full.js
+++ b/packages/tasit-action/src/erc721/ERC721Full.js
@@ -1,5 +1,8 @@
 import Contract from "../contract/Contract";
-import abi from "../../../tasit-contracts/abi/MyERC721Full.json";
+import TasitContracts from "tasit-contracts";
+const { local } = TasitContracts;
+const { MyERC721Full } = local;
+const { abi } = MyERC721Full;
 
 export default class ERC721Full extends Contract {
   constructor(address, wallet) {

--- a/packages/tasit-action/src/erc721/ERC721Full.test.js
+++ b/packages/tasit-action/src/erc721/ERC721Full.test.js
@@ -1,6 +1,6 @@
 import ERC721Full from "./ERC721Full";
 
-import TasitContracts from "../../../tasit-contracts/dist";
+import TasitContracts from "tasit-contracts";
 const { local: localContracts } = TasitContracts;
 const { MyERC721Full, SampleContract } = localContracts;
 const { address: ERC721_FULL_ADDRESS } = MyERC721Full;

--- a/packages/tasit-action/src/erc721/Estate.js
+++ b/packages/tasit-action/src/erc721/Estate.js
@@ -1,9 +1,11 @@
 import Contract from "../contract/Contract";
-import estateRegistryABI from "../../../tasit-contracts/abi/EstateRegistry.json";
+import TasitContracts from "tasit-contracts";
+const { local } = TasitContracts;
+const { EstateRegistry } = local;
+const { abi } = EstateRegistry;
 
 export default class DecentralandEstate extends Contract {
   constructor(address, wallet) {
-    const abi = estateRegistryABI;
     super(address, abi, wallet);
   }
 }

--- a/packages/tasit-action/src/erc721/Estate.test.js
+++ b/packages/tasit-action/src/erc721/Estate.test.js
@@ -1,5 +1,5 @@
 import Estate from "./Estate";
-import TasitContracts from "../../../tasit-contracts/dist";
+import TasitContracts from "tasit-contracts";
 const { local: localContracts } = TasitContracts;
 const { EstateRegistry } = localContracts;
 const { address: ESTATE_ADDRESS } = EstateRegistry;

--- a/packages/tasit-action/src/erc721/Land.js
+++ b/packages/tasit-action/src/erc721/Land.js
@@ -1,9 +1,11 @@
 import Contract from "../contract/Contract";
-import landRegistryABI from "../../../tasit-contracts/abi/LANDRegistry.json";
+import TasitContracts from "tasit-contracts";
+const { local } = TasitContracts;
+const { LANDRegistry } = local;
+const { abi } = LANDRegistry;
 
 export default class Land extends Contract {
   constructor(address, wallet) {
-    const abi = landRegistryABI;
     super(address, abi, wallet);
   }
 }

--- a/packages/tasit-action/src/erc721/Land.test.js
+++ b/packages/tasit-action/src/erc721/Land.test.js
@@ -1,5 +1,5 @@
 import Land from "./Land";
-import TasitContracts from "../../../tasit-contracts/dist";
+import TasitContracts from "tasit-contracts";
 const { local: localContracts } = TasitContracts;
 const { LANDProxy } = localContracts;
 const { address: LAND_PROXY_ADDRESS } = LANDProxy;

--- a/packages/tasit-action/src/ethers.test.js
+++ b/packages/tasit-action/src/ethers.test.js
@@ -1,6 +1,6 @@
 import { ethers } from "ethers";
 ethers.errors.setLogLevel("error");
-import TasitContracts from "../../tasit-contracts/dist";
+import TasitContracts from "tasit-contracts";
 const { local: localContracts } = TasitContracts;
 const { SampleContract } = localContracts;
 const {

--- a/packages/tasit-action/src/marketplace/Decentraland.js
+++ b/packages/tasit-action/src/marketplace/Decentraland.js
@@ -1,9 +1,11 @@
 import Contract from "../contract/Contract";
-import marketplaceABI from "../../../tasit-contracts/abi/Marketplace.json";
+import TasitContracts from "tasit-contracts";
+const { local } = TasitContracts;
+const { Marketplace } = local;
+const { abi } = Marketplace;
 
 export default class Decentraland extends Contract {
   constructor(address, wallet) {
-    const abi = marketplaceABI;
     super(address, abi, wallet);
   }
 }

--- a/packages/tasit-action/src/marketplace/Decentraland.test.js
+++ b/packages/tasit-action/src/marketplace/Decentraland.test.js
@@ -1,5 +1,5 @@
 import Decentraland from "./Decentraland";
-import TasitContracts from "../../../tasit-contracts/dist";
+import TasitContracts from "tasit-contracts";
 const { local: localContracts } = TasitContracts;
 const { Marketplace } = localContracts;
 const { address: MARKETPLACE_ADDRESS } = Marketplace;

--- a/packages/tasit-action/src/testHelpers/helpers.js
+++ b/packages/tasit-action/src/testHelpers/helpers.js
@@ -1,7 +1,7 @@
 import { expect } from "chai";
 import { ethers } from "ethers";
 import ProviderFactory from "../ProviderFactory";
-import developmentConfig from "../config/default.js";
+import developmentConfig from "../config/default";
 
 // Note:  Using dist file because babel doesn't compile node_modules files.
 // Any changes on src should be followed by compilation to avoid unexpected behaviors.

--- a/packages/tasit-action/src/testHelpers/mochaSetup.js
+++ b/packages/tasit-action/src/testHelpers/mochaSetup.js
@@ -12,7 +12,7 @@ import actionHelpers from "./helpers";
 global = Object.assign(global, actionHelpers);
 
 import ConfigLoader from "../ConfigLoader";
-import config from "../config/default.js";
+import config from "../config/default";
 ConfigLoader.setConfig(config);
 
 // Global hooks

--- a/packages/tasit-contracts/package.json
+++ b/packages/tasit-contracts/package.json
@@ -2,6 +2,7 @@
   "name": "tasit-contracts",
   "version": "0.0.11",
   "description": "",
+  "main": "dist/index.js",
   "keywords": [
     "ethereum",
     "dapp",

--- a/packages/tasit-identity-contract/package.json
+++ b/packages/tasit-identity-contract/package.json
@@ -27,7 +27,8 @@
   "dependencies": {
     "@babel/runtime": "^7.1.5",
     "ethers": "^4.0.27",
-    "tasit-action": "^0.0.11"
+    "tasit-action": "^0.0.11",
+    "tasit-contracts": "^0.0.11"
   },
   "devDependencies": {
     "@babel/cli": "^7.1.5",

--- a/packages/tasit-identity-contract/src/GnosisSafe.js
+++ b/packages/tasit-identity-contract/src/GnosisSafe.js
@@ -4,7 +4,7 @@ const { ERC20Detailed } = ERC20;
 const { ERC721Full } = ERC721;
 import GnosisSafeUtils from "./GnosisSafeUtils";
 import ActionUtils from "tasit-action/dist/contract/Utils.js";
-import TasitContracts from "../../tasit-contracts/dist";
+import TasitContracts from "tasit-contracts";
 const { local } = TasitContracts;
 const { GnosisSafe: GnosisSafeInfo, MyERC20Full, MyERC721Full } = local;
 const { abi: gnosisSafeABI } = GnosisSafeInfo;

--- a/packages/tasit-identity-contract/src/GnosisSafe.test.js
+++ b/packages/tasit-identity-contract/src/GnosisSafe.test.js
@@ -4,7 +4,7 @@ const { ERC20Full } = ERC20;
 const { ERC721Full } = ERC721;
 import GnosisSafe from "./GnosisSafe";
 import Account from "../../tasit-account/dist";
-import TasitContracts from "../../tasit-contracts/dist";
+import TasitContracts from "tasit-contracts";
 const { local } = TasitContracts;
 const { GnosisSafe: GnosisSafeInfo, MyERC20Full, MyERC721Full } = local;
 const { address: GNOSIS_SAFE_ADDRESS } = GnosisSafeInfo;

--- a/packages/tasit-sdk/package.json
+++ b/packages/tasit-sdk/package.json
@@ -32,7 +32,8 @@
   "dependencies": {
     "@babel/runtime": "^7.1.5",
     "tasit-account": "^0.0.11",
-    "tasit-action": "^0.0.11"
+    "tasit-action": "^0.0.11",
+    "tasit-contracts": "^0.0.11"
   },
   "devDependencies": {
     "@babel/cli": "^7.1.5",
@@ -42,7 +43,6 @@
     "@babel/register": "^7.0.0",
     "chai": "^4.2.0",
     "ethers": "^4.0.27",
-    "mocha": "^6.0.1",
-    "tasit-contracts": "^0.0.11"
+    "mocha": "^6.0.1"
   }
 }

--- a/packages/tasit-sdk/package.json
+++ b/packages/tasit-sdk/package.json
@@ -42,6 +42,7 @@
     "@babel/register": "^7.0.0",
     "chai": "^4.2.0",
     "ethers": "^4.0.27",
-    "mocha": "^6.0.1"
+    "mocha": "^6.0.1",
+    "tasit-contracts": "^0.0.11"
   }
 }

--- a/packages/tasit-sdk/src/helpers/DecentralandUtils.js
+++ b/packages/tasit-sdk/src/helpers/DecentralandUtils.js
@@ -1,5 +1,5 @@
 import { ethers } from "ethers";
-import TasitContracts from "../../../tasit-contracts/dist";
+import TasitContracts from "tasit-contracts";
 // ABI is the same among the networks. The ABI from local will work with tesnet.
 const { local } = TasitContracts;
 const { Marketplace } = local;

--- a/packages/tasit-sdk/src/testHelpers/mochaSetup.js
+++ b/packages/tasit-sdk/src/testHelpers/mochaSetup.js
@@ -24,7 +24,7 @@ ConfigLoader.setConfig(config);
 const { provider: configProvider } = config;
 const { network } = configProvider;
 
-import TasitContracts from "../../../tasit-contracts/dist";
+import TasitContracts from "tasit-contracts";
 const { local, goerli, ropsten } = TasitContracts;
 let blockchain;
 if (network === "goerli") blockchain = goerli;

